src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -276,10 +279,9 @@ public void bannedUserCanNotEnterTheChat() {
       Assertion.assertTrue(chatUserToBeBanned.isPermissionsErrorTitleDisplayed(), "PERMISSION ERROR IS NOT DISPLAYED");
     } catch (InterruptedException e) {
       e.printStackTrace();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)